wrappers/jira_client.py

```diff
@@ -25,23 +28,31 @@ def convert_seconds_to_jira_time(seconds):
 
         return " ".join(jira_time) if jira_time else "0s"
 
-    def __init__(self, config):
+    def __init__(self, config: dict):
         self.jira_url = config.get('jira_url')
-        self.tempo_api = config.get('tempo_api')
-        self.tempo_user = config.get('tempo_user')
-        self.tempo_password = config.get('tempo_password')
-        self.session = self.init_session()
-
-    def init_session(self):
+        self.tempo_url = config.get('tempo_url')
+        self.worklog_url = urljoin(self.tempo_url, 'worklogs')
+        self.jira_api_token = config.get('jira_api_token')
+        self.tempo_api_token = config.get('tempo_api_token')
```

Comment:
This sounds wrong: api tokens should not come from the config but from the env vars